<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbLoad">
       <property name="text">
        <string>Ucitaj gradove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gradovi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listCities"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbBuild">
       <property name="text">
        <string>Izgradi puteve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Putevi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="teBuild"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
